src/post/Write/EditorActions.js

```diff
@@ -19,23 +20,36 @@ export const CREATE_POST_ERROR = '@editor/CREATE_POST_ERROR';
 
 const requiredFields = 'parentAuthor,parentPermlink,author,permlink,title,body,jsonMetadata'.split(',');
 
+const getPermlink = (author, permlink) =>
```

Comment:
Did you get the algorithm from steemit.com rep? If not please do so and if you did add a comment to the link of the source code you borrow the algorithm from so we can have it for future reference (if they changed it etc)

Also, since it's common better to add it to `src/helpers/steemitHelpers.js` like the way I did with commentPermlink.